src/main/java/com/xceptance/xlt/engine/util/IncludedFilesResolver.java

```diff
@@ -306,10 +306,9 @@ private static void checkForCyclicInclude(final List<String> alreadyMarked, fina
      */
     private static Properties getPropertiesFromFile(final FileObject current) throws IllegalStateException
     {
```

Comment:
What about to simply delegate to `PropertiesUtils.loadProperties(FileObject)` inside the try-block?

Reply:
Good idea! Done.